console/console-init/ui/src/modules/msg-and-iot/dailogs/components/IoTReview/IoTReview.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React, { useState } from "react";
+import AceEditor from "react-ace";
+import {
+  PageSection,
+  Title,
+  Grid,
+  GridItem,
+  Tooltip,
+  TooltipPosition,
+  Button,
+  ButtonVariant,
+  PageSectionVariants
+} from "@patternfly/react-core";
+import { StyleSheet, css } from "@patternfly/react-styles";
+import { OutlinedCopyIcon } from "@patternfly/react-icons";
+
+export interface IIoTReviewProps {
+  name?: string;
+  namespace: string;
+  isEnabled: boolean;
+}
+
+const Style = StyleSheet.create({
+  left_padding: {
+    paddingLeft: 32
+  },
+  bottom_padding: {
+    paddingBottom: 16
+  }
+});
+
+export const IoTReview: React.FunctionComponent<IIoTReviewProps> = ({
+  name,
+  namespace,
+  isEnabled
+}) => {
+  const [isCopied, setIsCopied] = useState<boolean>(false);
+  return (
+    <PageSection variant={PageSectionVariants.light}>
+      <Title size="3xl" style={{ marginBottom: 32 }}>
+        Review your configuration
+      </Title>
+      <Title size="xl" style={{ marginBottom: 32 }}>
+        {" "}
+        Review the information below and Click Finish to create the new iot
+        project. Use the Back button to make changes.
+      </Title>
+      <Grid>
```

Comment:
According to me, this component is a bit complex, we should try to divide this component.